### PR TITLE
Week 4 Homework Submission

### PR DESCRIPTION
**Pull request by seesarahcode**
Submits homework solutions for week 4.

### CODE REVIEW

File: week2/homework/questions.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 Please Read The Chapters on:
```
Comment (bethanyr) [inline week2/homework/questions.txt:2]: I don't see any answers for the questions for week 2.

File: week3/homework/questions.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (bethanyr) [inline week3/homework/questions.txt:1]: I don't see any answers for week 3 questions for homework.

**Comment by bethanyr** [comment]
I have reviewed week2, week3 and week4 homework. You are missing week 2 questions and week 3 questions - can you submit those in your next pull request to get credit?  Week 4 homework looks good and the code for week2 and week 3 looks good.

**Comment by seesarahcode** [comment]
Hi Bethany,

Whoops, sorry about that! I think I accidentally deleted them at some point
while trying to clean up my files, but I just submitted a pull request to
add them back in.

Glad to hear that I'm doing okay with the assignments--I appreciate the
feedback!

Warm regards,

Sarah W.

On Thu, Feb 6, 2014 at 5:42 PM, Bethany Rentz notifications@github.comwrote:

> I have reviewed week2, week3 and week4 homework. You are missing week 2
> questions and week 3 questions - can you submit those in your next pull
> request to get credit? Week 4 homework looks good and the code for week2
> and week 3 looks good.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/UWE-Ruby/RubyWinter2014/pull/70#issuecomment-34396453
> .

## 

__

Sarah C. Wheeler
M.A. in Women's Studies
sarahcwheeler@gmail.com

**Pull request closed without merging**